app/src/main/res/values/strings.xml

```diff
@@ -186,6 +187,11 @@
     </item>
   </plurals>
   <string name="bar_separator">\u0020|\u0020</string>
+  <!-- Activity Titles -->
```

Comment:
This should be name as file, look for other for reference.

Reply:
Renamed to `<!-- DeveloperOptionsActivity -->`.